pkg/handlers/config.go

```diff
@@ -64,6 +64,69 @@ type CurrentAppConfigResponse struct {
 	ConfigGroups []kotsv1beta1.ConfigGroup `json:"configGroups"`
 }
 
+type DownloadAppConfigResponse struct {
+	Success bool   `json:"success"`
+	Error   string `json:"error,omitempty"`
+}
+
+func (h *Handler) DownloadFileFromConfig(w http.ResponseWriter, r *http.Request) {
```

Comment:
since this was renamed, i think you'll need to run `make mock` too.

Reply:
yup, done